### ISSUE TITLE
Scene settings are fluctuating randomly and are not saved

### ISSUE THREAD

**Issue by ChrisPrefect** [dresden-elektronik/phoscon-app-beta#65]
I made a video about the issue:

[Phoscon App - Google Chrome 2020-04-14 23-48-18.zip](https://github.com/dresden-elektronik/phoscon-app-beta/files/4477942/Phoscon.App.-.Google.Chrome.2020-04-14.23-48-18.zip)

It also looks like the connection is really unstable. After only about 60 devices added, commands often get not trough to all lights and some lights are shown as "offline" or they don't react to brightness or temperature changes in the app. 

This is all on one floor and lights are online every few meters, so the mesh network should reach every device without a problem.

ConBee II stick on a Raspi3B+ with USB extension cable. Zigbee channel 25, Unifi-WIFI on channel 1.

**Comment by YKO-de** [comment]
Does the problem still exist, or has it been solved in the meantime? 
If it still exists. From which manufacturer are the lamps.

**Comment by ChrisPrefect** [comment]
Yes it still exists. They are all Tradfri-Devises. Scenes are barely working, color temperature is not saved most of the time, lamps that should be off don't go off in scenes (was declared as nofix in another issue?) etc.

**Comment by SwoopX** [comment]
We may should move that to the API repo. It feeld more right there. @YKO-de if you could be so kind 😉

**Comment by ChrisPrefect** [comment]
Any updates on this?
